docs/authoring/working_with_teams.md

```diff
@@ -0,0 +1,514 @@
+        1. How will the templates be grouped or bundled for different recipients?
+        1. Are any documents produced in a mirrored form (for example, for each spouse) or copied for 
+           multiple named parties (for example, one per child?)
+        1. Will any ancillary documents need to be created that aren't already in existence, like:
+                * Addenda for overflow text
+                * Affidavits
+                * Motions
+1. What happens after the user completes the automated steps?
+        * How will the document be signed or notarized?
+        * How will it be served on the other party, if applicable?
+        * How will copies be provided to other parties, such as designated agents, hospitals, or others?
+        * How will it be filed in court, if applicable?
+        * What does the user need to know about the possible outcomes and decisions that will be made
+          based on their document?
+1. Should any parts of the process be left **un**-automated?
+1. What compliance or regulation needs to be kept in mind?
+
+One approach to this is to interview someone who currently completes the process. Have them pretend
+that you are the client and ask questions as they walk you through filling in the form or gathering the information
+needed to complete the document.
+
+Interview key stakeholders to get answers. Write down the responses and make your own observations. 
+Share your summary with the stakeholders to make sure they agree with your assessment.
+
+## Mark up the templates
+
+Annotate the documents so you know where fields will go. Typically, this 
+includes the following:
+
+1. Names of parties or agents, other participants, locations, and organizations
+1. Pronouns, quantities, relationships, and titles
+1. Language that **depends** on quantity, relationship, or title
+1. Dates
+1. Dollar figures and financial terms
+1. Conditional clauses
+1. Signatures
+
+Review the annotated document with the client to check your understanding about what parts of
+the document need to be part of the interview. Some information may be static
+that you don't expect. Some may need to be dynamic.
+
+Preferences will vary, for example, about whether to make pronouns depend on
+the parties named in the document or to use gender neutral language.
+
+## Share drafts early and often for feedback
+
+### Consider using flowcharts, wireframes or low fidelity prototypes to communicate your vision
+
+You can use a tool like:
+
+1. Figma
+1. Penpot
+1. Flowchart.fun
+
+or even PowerPoint or pen and paper sketches to communicate an early vision
+of how the tool will work. This should include key branching logic.
+
+This isn't necessary in every straightforward automation project or if
+it would take less time to build a functional basic automation than to 
+turn it into a flowchart or wireframes. The [Weaver](https://github.com/SuffolkLITLab/docassemble-ALWeaver) helps reach the prototype stage
```

Comment:
I know folks are supposed to install the weaver to use on their own servers. Is this the right entry point for that? If so, do we want to link to the docs on how to install the weaver and why? If not, do we want to give them a link to where they can use the weaver on our server instead of the one to the repo?

Reply:
Not in scope since the page you're describing doesn't exist. I also wanted to avoid a cross reference here since I feel the proper linking method may be in flux.